File: Ask2/ask2.py
name = '' #Όνομα
gender = 'Β' #Φύλο
b_sugar = -1 #Σάκχαρο
b_s_boys = 0 #Αγόρια με σάκχαρο εκτός φυσιολογικών ορίων
b_s_girls = 0 #Κορίτσια με σάκχαρο εκτός φυσιολογικών ορίων
for i in range (1,90):
    while (gender != 'Α' and gender!='Κ')and(b_sugar<=0) :
        name = input('Δώσε ονοματεπώνυμο μαθητή \n >>> ')
        gender = input('Δώσε "Α" για αγόρι και "Κ" για κορίτσι \n >>> ')
        b_sugar = input('Δώσε περιεκτικότητα σακχάρου στο αίμα \n >>> ')
    if b_sugar>110 and b_sugar<70:
        print('Υψηλή περιεκτικότητα ',name,' ',gender,' ',b_sugar,'mg/dl.')
        if gender == 'Α': 
            b_s_boys+=1
        else:
            b_s_girls+=1
print('Αγόρια με σάκχαρο εκτός των ορίων: ',b_s_boys)
print('Κορίτσια με σάκχαρο εκτός των ορίων: ',b_s_girls)

File: Ask2/ask2.c
#include <string.h>
#include <stdio.h>

char name[100] = "None"; // Όνομα
char gender = 'B';       // Φύλο
float b_sugar = -1;      // Σάκχαρο
int b_s_boys = 0; //Αγόρια με σάκχαρο εκτός ορίων
int b_s_girls = 0; //Κορίτσια με σάκχαρο εκτός ορίων
int main() {
    for (int i = 1; i <= 90; i++) {
        do {
            printf("Enter the student's name \n");
            fgets(name, sizeof(name), stdin);
            int len = strlen(name);
            if (name[len - 1] == '\n') {
                name[len - 1] = '\0';
            }
            printf("Enter A for by or K for girl \n");
            scanf("%c",gender);
            printf("Enter blood sugar: /n");
            scanf("%f",&b_sugar);
        } while (gender != 'A' && gender != 'K' && b_sugar > 0);
        if (b_sugar > 110 || b_sugar <70) {
            printf("%s %s %f Abnormal Blood Sugar \n",name,gender,b_sugar);
            if (gender == 'A') {
                b_s_boys++;
            }else {
                b_s_girls++;
            }
        }
    }
    printf("Boys with abnormal blood sugar: %d \n",b_s_boys);
    printf("Girls with abnormal blood sugar: %d \n",b_s_girls);
    return 0;
}
